jenkins/job-configs/jenkins-daily-maintenance.yaml

```diff
@@ -16,6 +16,12 @@
             docker rm -v "${containers_to_remove[@]:+${containers_to_remove[@]}}" || true
             docker rmi $(docker images -q -f 'dangling=true') || true
             docker run -v /var/run/docker.sock:/var/run/docker.sock -v /var/lib/docker:/var/lib/docker --rm martin/docker-cleanup-volumes
+            # Clean up leaked VMs from node e2e tests
+            gcloud -q compute instances list \
```

Comment:
Cleaning up after the node e2e tests is the right solution (and we do that when they don't timeout), though fixing the logic to properly clean up on timeout is a bit trickier. Also, there will always occasionally be leaked resources, though hopefully that'll be the exception.